Created screens for feed page and post modal

pritam-kr opened the pull request:
- Created modal component for the post
- feed page

![Screenshot (156)](https://user-images.githubusercontent.com/84632214/163703239-23580f4e-c3a3-4da1-97e0-5d083823cd81.png)

pritam-kr committed:
Created screens for feed page and post modal

pritam-kr committed:
Update Login.jsx

gopalbbharadva commented on src/Pages/Profile/Profile.jsx:
```diff
@@ -39,12 +43,28 @@ const Profile = () => {
           </div>
+
+          
+             {/* <div className="post-form-modal fixed top-0 left-0 right-0 bottom-0 flex justify-center items-start w-full ">
+              <div className="post-form-wrapper mx-3">
+                <p className="p-6 border-b-2"><FaIcons.FaTimes className="icons"/></p>
+                <form className="post-form p-2">
+                      <input className="input mt-2" type="text" placeholder="Write Caption"  />
+                      <textarea className="input mt-2" type="text" placeholder="What's Happening..."></textarea>
+
+                      <div className="flex gap-2 mt-2 mb-2">
+                        <input type="file" className="input file-upload w-3/4 cursor-pointer"/> <button className="btn w-3/4 text-caption">Post</button>
+                      </div>
+
+                </form>
+              </div>
+            </div> */}
```
Remove comments if not needed.

pritam-kr replied:
its for modal

gopalbbharadva reviewed (approved):
UI looks clean, Good to go.

kiranpratyush commented on src/Pages/AuthPage/Style.css:
```diff
@@ -3,7 +3,7 @@
      border-radius: 2px;
      outline: none;
-     background-color: var(--lite-blue);
+     /* background-color: var(--lite-blue); */
```
We can remove comment if not necessary

kiranpratyush reviewed (approved):
The code looks clean. We can use prettier to format if not done

KoushikKintali commented on src/Components/PostCard/PostCard.jsx:
```diff
@@ -1,51 +1,49 @@
+      <div className="post-thumbnail-wrapper my-2 p-0 ">
+        <img
+          src="https://avatars.githubusercontent.com/u/84632214?v=4"
+          className=""
```
if className is not required, u can remove that

pritam-kr replied:
ok

KoushikKintali commented on src/Pages/Profile/Profile.jsx:
```diff
@@ -39,12 +43,28 @@ const Profile = () => {
+          
+             {/* <div className="post-form-modal fixed top-0 left-0 right-0 bottom-0 flex justify-center items-start w-full ">
+              <div className="post-form-wrapper mx-3">
+                <p className="p-6 border-b-2"><FaIcons.FaTimes className="icons"/></p>
```
remove the commented code if not needed

pritam-kr replied:
ok

pritam-kr committed:
Create Three modal

- modal for followers and following, post
- connected all modal through context

pritam-kr commented:
Created three modal
- post, followers, following 
- to operate these modal created context for this

![ newwww](https://user-images.githubusercontent.com/84632214/163758783-ee8411a5-8d77-46ca-bb2c-678bbccea604.gif)
